### PR TITLE
Enhance OpenSCAP page

### PR DESCRIPTION
**Pull request by paususe**
# Description

While preparing the release notes for the next version, I noticed we are lacking a lot of details  in the OpenSCAP section. This PR adds them:
- Scanner package names
- Content package names
- How to list files in DEB-based operating systems
- How to find the profiles provided by each content file
- How to find and apply remediations (bash and Ansible)

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/administration/pages/openscap.adoc [hunk 1]
```diff
@@ -42,32 +42,32 @@ For more information about the {suse} Security Team, see  https://www.suse.com/s
 
 Before you begin, you need to prepare your client systems for SCAP scanning.
 
-For traditional and Salt clients, install the ``openscap-utils`` package before you begin.
+For traditional and Salt clients, install the OpenSCAP scanner and the SCAP Security Guide (content) packages before you begin. Depending on the operating system, these packages are included either on the base operating system, or in the {productname} Client Tools.
 
-Use this command to determine the location of the appropriate SCAP files.
-Take a note of the file paths for performing the scan:
+The table below listst the packages you need depending on your client operating system:
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc:47]:
```suggestion
The table below lists the packages you need depending on your client operating system:
```

File: modules/administration/pages/openscap.adoc [hunk 2]
```diff
@@ -42,32 +42,32 @@ For more information about the {suse} Security Team, see  https://www.suse.com/s
...
+| openSUSE | openscap-utils | scap-security-guide
+| RHEL | openscap-utils | scap-security-guide-redhat
+| CentOS | openscap-utils | scap-security-guide-redhat
+| Oracle Linux | openscap-utils | scap-security-guide-redhat
+| Ubuntu | libopenscap8 | scap-security-guide-ubuntu
+| Debian | libopenscap8 | scap-security-guide-debian
+|===
 
 
+RHEL 7 and compatible provide a `scap-security-guide` package which contains outdated content. You are advised to use the `scap-security-guide-redhat` package you will find in the {productname} Client Tools.
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc:63]:
```suggestion
RHEL 7 and compatible systems provide a [package]``scap-security-guide`` package, which contains outdated contents.
You are advised to use the [package]``scap-security-guide-redhat`` package you will find in the {productname} Client Tools.
```

File: modules/administration/pages/openscap.adoc [hunk 3]
```diff
@@ -79,25 +79,85 @@ If your system becomes damaged through the use of custom content files, you migh
 ====
 
 When you have created your content files, you need to transfer the file to the client.
-You can do this in the same way as you move any other file, using physical storage media, or across a network with [command]``ftp`` or [command]``scp``.
+You can do this in the same way as you move any other file, using physical storage media, or across a network with Salt (e. g. https://docs.saltproject.io/en/latest/ref/cli/salt-cp.html[salt-cp] or the https://docs.saltproject.io/en/latest/ref/file_server/index.html[Salt File Server]), [command]``ftp`` or [command]``scp``.
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc:82]:
```suggestion
You can do this in the same way as you move any other file, using physical storage media, or across a network with Salt (for example, https://docs.saltproject.io/en/latest/ref/cli/salt-cp.html[salt-cp] or the https://docs.saltproject.io/en/latest/ref/file_server/index.html[Salt File Server]), [command]``ftp`` or [command]``scp``.
```

File: modules/administration/pages/openscap.adoc [hunk 4]
```diff
@@ -79,25 +79,85 @@ If your system becomes damaged through the use of custom content files, you migh
...
 For more information, see xref:administration:custom-channels.adoc[].
 
+== Find OpenSCAP profiles
+
+Different operating systems make available different OpenSCAP content files and profiles. One content file may contain more than one profile.
+
+On RPM-based operating systems, use this command to determine the location of the available SCAP files:
+
+----
+$ rpm -ql <scap-security-guide-package-name-from-table>
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc:95]:
```suggestion
rpm -ql <scap-security-guide-package-name-from-table>
```

File: modules/administration/pages/openscap.adoc [hunk 5]
```diff
@@ -79,25 +79,85 @@ If your system becomes damaged through the use of custom content files, you migh
...
+----
+
+On DEB-based operating systems, use this command to determine the location of the available SCAP files:
+
+----
+$ dpkg -L <scap-security-guide-package-name-from-table>
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc:101]:
```suggestion
dpkg -L <scap-security-guide-package-name-from-table>
```

File: modules/administration/pages/openscap.adoc [hunk 6]
```diff
@@ -79,25 +79,85 @@ If your system becomes damaged through the use of custom content files, you migh
...
+----
+
+When you have identified one SCAP content file that suits your needs, list what are the available profiles on the client:
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc:104]:
```suggestion
When you have identified one SCAP content file that suits your needs, list profiles available on the client:
```

File: modules/administration/pages/openscap.adoc [hunk 7]
```diff
@@ -79,25 +79,85 @@ If your system becomes damaged through the use of custom content files, you migh
...
+
+----
+$ oscap info ./usr/share/xml/scap/ssg/content/ssg-sle15-ds-1.2.xml
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc:107]:
```suggestion
oscap info ./usr/share/xml/scap/ssg/content/ssg-sle15-ds-1.2.xml
```

File: modules/administration/pages/openscap.adoc [hunk 8]
```diff
@@ -79,25 +79,85 @@ If your system becomes damaged through the use of custom content files, you migh
...
+Document type: Source Data Stream
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc:108]: That outputs:

----

File: modules/administration/pages/openscap.adoc [hunk 9]
```diff
@@ -79,25 +79,85 @@ If your system becomes damaged through the use of custom content files, you migh
...
 You can also use the {productname} API to schedule regular scans.
 
 .Procedure: Running an Audit Scan from the {webui}
 . In the {productname} {webui}, navigate to menu:Systems[Systems List] and select the client you want to scan.
 . Navigate to the [guimenu]``Audit`` tab, and the [guimenu]``Schedule`` subtab.
-. In the [guimenu]``Path to XCCDF Document`` field, enter the path to the XCCDF content file on the client.
-  For example: [path]``/usr/share/openscap/scap-yast2sec-xccdf.xml``
+. In the [guimenu]``Path to XCCDF Document`` field, enter the parameters for the SCAP template and profile you want to use on the client.
+  For example:
+  ```
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc:156]:
```suggestion
----
```

File: modules/administration/pages/openscap.adoc [hunk 10]
```diff
@@ -79,25 +79,85 @@ If your system becomes damaged through the use of custom content files, you migh
...
+  Command: /usr/bin/oscap xccdf eval
+  Command-line arguments: --profile xccdf_org.ssgproject.content_profile_standard
+  Path to XCCDF document: /usr/share/xml/scap/ssg/content/ssg-sle15-ds-1.2.xml
+  ```
```
Comment (keichwa) [inline modules/administration/pages/openscap.adoc:160]:
```suggestion
----

```

**Review by keichwa — APPROVED** [review]
some cosmetic changes

File: modules/administration/pages/openscap.adoc (hunk 8 above)
Comment (paususe) [reply modules/administration/pages/openscap.adoc:109]: I don't know what you mean, it looks fine here :-?

image

Comment (keichwa) [reply modules/administration/pages/openscap.adoc:109]: I wanted to separate the command from the output of the command (two blocks).  But your way is also fine.

**Review by keichwa — APPROVED** [review]
Pau, do you have commit/merge right in this repo?  If not, I can merge it for you.

Nearly forgotten, we will also need an entry in the top-level `.changelog`.  I'll try to add one here.

**Pull request merged**